tooling/cli/src/build.rs

```diff
@@ -198,6 +199,43 @@ pub fn command(mut options: Options) -> Result<()> {
 
   interface.build(interface_options)?;
 
+  if let Some(before_bundle) = config_.build.before_bundle_command.clone() {
```

Comment:
I think we should only run `before_bundle_command` after the `if config_.tauri.bundle.active {` block, and only if `package_types` isn't empty.

Reply:
maybe `afterCargoBuild` is better?

Reply:
The hook is only needed because the bundler uses the output binary without letting the user modify it; an `afterCargoBuild` doesn't make sense IMO because if you need to modify the binary and you're not bundling it, you can just run your script after `tauri build`.